app/src/main/java/com/architectcoders/aacboard/model/ArasaacPictogram.kt

```diff
@@ -0,0 +1,19 @@
+package com.architectcoders.aacboard.model
+
+import com.google.gson.annotations.SerializedName
+
+data class ArasaacPictogram(
+    @SerializedName("_id")
+    val id: Int,
+    val keywords: List<Keyword>,
+    val sex: Boolean,
+    val violence: Boolean
+) {
+    fun getUrl(): String {
```

Comment:
possibility of using property extension instead of function for the url property